keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -245,35 +247,35 @@ spec:
   provisioner: [4]
   releasePolicy: [5]
   accessMode: [6]
-  protocol: [7]
-  bucketClassName: [8]
-  bucketRef: [9]
+  bucketClassName: [7]
+  bucketRef: [8]
     name:
     namespace:
     uuid:
     resourceVersion:
-  secretRef: [10]
+  secretRef: [9]
     name:
     namespace:
-  azureBlobIdentity: [11]
+  azureBlobIdentity: [10] // TODO break into auth and conn
```

Comment:
If we remove `protocol` how do we know which of the _xxIdentity_ fields to use, in the case of a driver supporting >1 protocol?

Reply:
The Bucket will specify the protocol (which is where we decided class matching would be a thing).